code/game/world.dm

```diff
@@ -22,14 +22,16 @@ GLOBAL_LIST_INIT(map_transition_config, MAP_TRANSITION_CONFIG)
 
 	TgsNew(new /datum/tgs_event_handler/impl, TGS_SECURITY_TRUSTED) // creates a new TGS object
 	log_world("World loaded at [time_stamp()]")
-	log_world("[GLOB.vars.len - GLOB.gvars_datum_in_built_vars.len] global variables")
+	log_world("[length(GLOB.vars) - length(GLOB.gvars_datum_in_built_vars)] global variables")
 	GLOB.revision_info.log_info()
-	load_admins(run_async=FALSE) // This better happen early on.
+	load_admins() // This better happen early on.
```

Comment:
Removed the argument here because it's already set in the proc itself:
```cs
/proc/load_admins(run_async = FALSE)
```

Reply:
Yes the arg is already set but given the circumstances of how obvious it needs to be that this proc cannot sleep, I would rather it stay there